pages/calendar/index.page.tsx

```diff
@@ -9,14 +13,44 @@ import WeeklyCalendar from "./components/Calendar/WeeklyCalendar";
 import CalendarSideBar from "./components/CalendarSideBar";
 import Header from "@/components/Header";
 import MetaData from "@/components/MetaData";
+import { useDateStore } from "@/store/useDateStore";
+import { calculateMonthDates } from "@/utils/calculateCalendarDates";
 
 import styles from "./Calendar.module.scss";
 
 export type CalendarType = "월" | "주" | "일";
```

Comment:
```suggestion
export type CalendarType = "월" | "주" | "일";
```
p4: 여러 곳에서 쓰일 것 같은 타입이라서 `type` 폴더 옮기면 나중에 유지보수하기 편할 것 같아요!